cli/pkg/bind/plugin/bind.go

```diff
@@ -221,8 +221,8 @@ func (b *BindOptions) parsePermissionClaim(claim string, accepted bool) error {
 	}
 
 	parsedClaim := apisv1alpha1.AcceptablePermissionClaim{}
-	resource := claimParts[0]
```

Comment:
Resource should be first `resource.group`. Why change this? Maybe doc/example was wrong

Reply:
I see, in that case indeed the docs string is wrong: https://github.com/kcp-dev/kcp/blob/main/cli/pkg/bind/cmd/cmd.go#L35

Reply:
Looking at the code and the docstring, this will only work for the core API anyway, no? The code assumes the format `X.Y`, but you can create permission claims for groups with a dot as well, so the parsing here would probably fail at those (e.g. `apibindings.apis.kcp.io`).

Reply:
Would `<group>/<resource>` be better?

Reply:
The syntax is fine, the parser just needs to take the element at index 0 as the resource name and concat the remaining elements into the group.